class-00/lab-a/starter-code/js/app.js

```diff
@@ -30,18 +30,20 @@ function makeRandom() {
 
 function displayPics(){
   while(viewed.length < 6){
-    var rando = makeRandom();
+    const rando = makeRandom();
     while(!viewed.includes(rando)){
       viewed.push(rando);
     }
   }
-  console.log(rando);
-  // TODO: In a sentence or two, explain why the previous line of code threw an error when we changed the variable declaration from `var to `let`.
+  
+  // DONE: In a sentence or two, explain why the previous line of code threw an error when we changed the variable declaration from `var to `let`.
   // PUT YOUR RESPONSE IN THIS COMMENT
+
+  //It was logging to something that didnt wasnt there

```

Comment:
I'm unsure what you are trying to say. Can you re-write your answer in a different way?